modules/ROOT/pages/_partials/endpoint-http-receive.adoc

```diff
@@ -0,0 +1,73 @@
+== HTTP Receive Settings
+
+These settings are a subset of those in xref:3.8@mule-runtime::http-listener-connector.adoc[HTTP Listener Connector].
+
+
+=== Global Config
+
+[%header,cols="3s,7a"]
+|===
+|Name |Description
+|Protocol |*HTTP* or *HTTPS* +
+Type: Radio-button-selected value
+Required: yes +
+Default: HTTP
+
+|Use Persistent Connections |&nbsp;+
```

Comment:
A space is missing here: `|&nbsp;+` should be `|&nbsp; +`
This is affecting the style and the `+` character at the end is being shown in the actual file.

Reply:
on line 12, could we "unpack" radio-button-selected? Value representing a radio-button selection?